src/mon/MgrStatMonitor.cc

```diff
@@ -93,6 +94,16 @@ void MgrStatMonitor::handle_conf_change(
     }
     enable_availability_tracking = newval;
   }
+
+  if (changed.count("pool_availability_update_interval")) {
+      std::scoped_lock l(lock);
+      dout(10) << __func__ << " pool_availability_update_interval config changed from " 
+             << pool_availability_update_interval << " to " 
+             << g_conf().get_val<bool>("pool_availability_update_interval")
+             << dendl;
+      
+      pool_availability_update_interval = g_conf().get_val<bool>("pool_availability_update_interval"); 
```

Comment:
hmm shouldn't it be "double" instead of "bool"?

for `get_val<bool>` both places here or am I missing something.